ray_on_golem/server/services/ray.py

```diff
@@ -196,6 +203,11 @@ async def _create_node(
             # TODO: check if node is a head node
             if not self._is_head_node_to_webserver_tunnel_running():
                 await self._start_head_node_to_webserver_tunnel()
+
+            # monitor activity state
+            self._node_monitoring_tasks[node_id] = asyncio.create_task(
```

Comment:
Use `create_task_with_logging` from golem-core here. Set proper `trace_id` too.

Reply:
add here and in other 2 palces